<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="23543" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:36 IST">
    <groups>
    </groups>
    <test name="validateNavigationBar" duration-ms="10468" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:23 IST">
      <class name="com.e2e.test.validateNavigationBar">
        <test-method status="PASS" signature="initialize()[pri:0, instance:com.e2e.test.validateNavigationBar@21282ed8]" name="initialize" is-config="true" duration-ms="10005" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="basePageNavigation()[pri:0, instance:com.e2e.test.validateNavigationBar@21282ed8]" name="basePageNavigation" duration-ms="392" started-at="2021-02-09T15:32:23 IST" finished-at="2021-02-09T15:32:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.e2e.test.validateNavigationBar@21282ed8]" name="tearDown" is-config="true" duration-ms="49" started-at="2021-02-09T15:32:23 IST" finished-at="2021-02-09T15:32:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.e2e.test.validateNavigationBar -->
    </test> <!-- validateNavigationBar -->
    <test name="ValidateTitle" duration-ms="11807" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:25 IST">
      <class name="com.e2e.test.validateTitle">
        <test-method status="PASS" signature="initialize()[pri:0, instance:com.e2e.test.validateTitle@2ed2d9cb]" name="initialize" is-config="true" duration-ms="10684" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="basePageNavigation()[pri:0, instance:com.e2e.test.validateTitle@2ed2d9cb]" name="basePageNavigation" duration-ms="67" started-at="2021-02-09T15:32:23 IST" finished-at="2021-02-09T15:32:24 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURED CO123URSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURED CO123URSES] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.e2e.test.validateTitle.basePageNavigation(validateTitle.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.e2e.test.validateTitle@2ed2d9cb]" name="tearDown" is-config="true" duration-ms="81" started-at="2021-02-09T15:32:24 IST" finished-at="2021-02-09T15:32:25 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.e2e.test.validateTitle -->
    </test> <!-- ValidateTitle -->
    <test name="Homepage" duration-ms="23542" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:36 IST">
      <class name="com.e2e.test.HomePage">
        <test-method status="PASS" signature="initialize()[pri:0, instance:com.e2e.test.HomePage@659499f1]" name="initialize" is-config="true" duration-ms="5371" started-at="2021-02-09T15:32:13 IST" finished-at="2021-02-09T15:32:18 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="FAIL" signature="basePageNavigation(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.e2e.test.HomePage@659499f1]" name="basePageNavigation" duration-ms="13796" started-at="2021-02-09T15:32:18 IST" data-provider="getData" finished-at="2021-02-09T15:32:32 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[nonrestricteduser@test.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[nonrestricteduser]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.e2e.pageObject.ForgotPassword.getForgotEmail(ForgotPassword.java:15)
at com.e2e.test.HomePage.basePageNavigation(HomePage.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="FAIL" signature="basePageNavigation(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.e2e.test.HomePage@659499f1]" name="basePageNavigation" duration-ms="3451" started-at="2021-02-09T15:32:32 IST" data-provider="getData" finished-at="2021-02-09T15:32:36 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[restricteduser@test.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[restricted user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.e2e.pageObject.ForgotPassword.getForgotEmail(ForgotPassword.java:15)
at com.e2e.test.HomePage.basePageNavigation(HomePage.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:425)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.e2e.test.HomePage@659499f1]" name="tearDown" is-config="true" duration-ms="59" started-at="2021-02-09T15:32:36 IST" finished-at="2021-02-09T15:32:36 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.e2e.test.HomePage -->
    </test> <!-- Homepage -->
  </suite> <!-- Suite -->
</testng-results>
